app/scripts/controllers/overview.js

```diff
@@ -64,6 +64,8 @@ angular.module('overview', ['stats'])
       'data': [],
     };
 
+    console.log(JSON.stringify(ClusterState, null, 2));
```

Comment:
please remove if not necessary any more :)
we could think about a global `DEBUG` state that enables useful logging